### PR TITLE
Changelog for disputes list page improvements.

### LINKED ISSUE #6355: Epic: WCPay Disputes – Optimise disputes list view
### Description
The goal of this epic is to make it easier to manage and respond to disputes from the disputes list view. 

Screenshot 2023-05-23 at 1 08 59 PM

dispute-list-1

Note – the above screenshot shows some other future improvements not scoped to this project. 

### Steps
- [x] #6661
- [x] #6667
- [x] #6702
- [x] #6722
- [x] #6692
- [x] Add Changelog entry for this epic's changes https://github.com/Automattic/woocommerce-payments/pull/6754
    - Remove/replace changelog entry for #6723 – `changelog/fix-6722-dispute-list-status-chip-colours`
- [x] Tracks / tracking
  - [x] #6747
- [x] Add testing instructions to this description for impacted flows

The above is a high-level summary of the changes in the design. These can be split off as separate issues. Can add / adapt if details change.

### Additional context (internal links)
- Discussion about 72h urgency criteria: p1688655548302669-slack-C03LA109BKM

## Testing flows

* Visit the Disputes listing page, Payments > Disputes
* Hide `Disputed on` column
	* Ensure that the `Disputed on` column is not visible.
	* Download the csv and see that the results are unchanged.
	* Ensure the `Disputed On` column can be enabled by using the options menu.
* `Action` column
	* See active disputes with `Respond` button.
	* Change filter from `Needs Response` to `All Disputes`.
	* See historical disputes with `See details` link in action column.
* `Response date` column
	* When due date is <= 72 hours from now, value is highlighted red with an icon and the text `N days left`
	* When due date is <= 24 hours from now, value is highlighted red with an icon and with the text `Last day today`
	* When due date is > 72 hours from now, value is a formatted date without highlighting.
* Status chip colours.
	* Ensure the status chip colours match the following criteria
		* red = needs response within 72h
		* yellow = needs response > 72h
		* green = won
		* blue = in progress / under review
		* grey = lost

### PR DESCRIPTION
**Pull request by shendy-a8c**
Fixes https://github.com/Automattic/woocommerce-payments/issues/6355

#### Changes proposed in this Pull Request

Changelog for changes in https://github.com/Automattic/woocommerce-payments/issues/6355.

#### Testing instructions

* Make sure changelog entry introduced in https://github.com/Automattic/woocommerce-payments/pull/6723/files#diff-00c2cb0c0f7e76b4def8746d5d84472f620b7360494104148b8f4cf3d4cef23d is changed to just a comment which won't turn into a real changelog entry.
* Make sure changelog entry is clear and covers all the changes in https://github.com/Automattic/woocommerce-payments/issues/6355.

ps: if you want to test generating the changelog, run this [command](https://github.com/Automattic/woocommerce-payments/blob/develop/.github/actions/process-changelog/action.yml#L67C11-L67C155).

-------------------

- [x] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [ ] Covered with tests (or have a good reason not to test in description ☝️)
- [ ] Tested on mobile (or does not apply)

**Post merge**

- [ ] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : _Add link here / 'QA Testing Not Applicable'_
- [ ] Add or update [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) and [testing instructions for critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows), if applicable.
- [ ] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable.

### CODE REVIEW

**Comment by botwoo** [comment]
### Test the build 

#### Option 1. Jetpack Beta

- Install and activate [Jetpack Beta](https://jetpack.com/download-jetpack-beta/).
- Use this build by searching for PR number `6754` or branch name `add/6355-changelog` in your-test.site/wp-admin/admin.php?page=jetpack-beta&plugin=woocommerce-payments          

#### Option 2. Jurassic Ninja - available for logged-in A12s

:rocket: [Launch a JN site with this branch](https://jurassic.ninja/create/?jetpack-beta&shortlived&nojetpack&woocommerce&branches.woocommerce-payments=add/6355-changelog) :rocket:

:information_source: Install this [Tampermonkey script](https://github.com/Automattic/woocommerce-payments/tree/develop/bin/wcpay-live-branches) to get more options.

---

Build info: 

- Latest commit: 82205f7ba86d235e1825e4e89239455dd32ae36d
- Build time: 2023-07-13 09:12:07 UTC

_Note: the build is updated when a new commit is pushed to this PR._

File: changelog/add-6355-changelog [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: update
+
+Improve disputes list page by removing Disputed on date column by default, add Action column with clear call to action button, highlight urgent disputes' due dates, and color code disputes' statuses.
```
Comment (Jinksi) [inline changelog/add-6355-changelog]: I've suggested minor changes to this, feel free to take or leave 👍

```suggestion
Improve disputes list page by hiding the "Disputed on" column by default, add an "Action" column with clear call to action button, highlight urgent disputes' due dates, and color code disputes' statuses.
```

**Review by Jinksi — APPROVED** [review]
LGTM!

I've tested with the `vendor/bin/changelogger write` command you mention in the description and the changelog correctly includes the new entry and removes the redundant entry.

File: changelog/add-6355-changelog (hunk 1 above)
Comment (shendy-a8c) [reply changelog/add-6355-changelog]: Take!

**Pull request merged**